tests/test_suspend_resume/src/main.xc

```diff
@@ -0,0 +1,68 @@
+// Copyright 2016-2021 XMOS LIMITED.
```

Comment:
It looks like this is a new file.  If so, the copyright date should be just '2021'.

Reply:
copy pasta. fixed